src/yang/ietf-bgp-rib.yang

```diff
@@ -330,6 +330,19 @@ submodule ietf-bgp-rib {
         "RFC 4360 - BGP Extended Communities Attribute";
     }
   }
+
+  grouping large-community-attributes {
+    description
+      "A grouping for all large community parameters.";
+
+    leaf-list large-community {
+      type bt:bgp-large-community-type;
+      description
+        "List of BGP large community attributes.";
+      reference
+        "RFC 8092 - BGP Large Communities Attribute";
```

Comment:
I know I am nitpicking - but in the spirit of trying to be consistent, I am moving towards using the following format for references

`"RFC <RFC number>: <The title of the draft>."`

The :  is not consistently used, but if we can agree on that as the format, I will update them in the models.

Reply:
I'll update here.  There's an existing issue open for fixing punctuation.  I suggest adding the format as a desired outcome to that issue.